client/account.go

```diff
@@ -0,0 +1,35 @@
+package client
+
+import (
+	"github.com/xyield/xrpl-go/model/client/account"
+)
+
+type Account interface {
+	// return result struct, fill xrpl response for warnings etc, error
+	GetAccountChannels(req *account.AccountChannelsRequest) (*account.AccountChannelsResponse, XRPLResponse, error)
+}
+
+type accountImpl struct {
+	Client Client
+}
+
+func (a *accountImpl) GetAccountChannels(req *account.AccountChannelsRequest) (*account.AccountChannelsResponse, XRPLResponse, error) {
+
+	err := req.Validate()
```

Comment:
Is it worth moving this into the send request function so we don't need to repeat it on every function?

Reply:
yeah makes sense, will move now

Reply:
Means we will leave all the validation testing to the test files in the model folder